protos/tracking_server/tracking_server.proto

```diff
@@ -0,0 +1,76 @@
+syntax = "proto3";
+
+package mavsdk.rpc.tracking_server;
+
+import "mavsdk_options.proto";
+
+option java_package = "io.mavsdk.tracking_server";
+option java_outer_classname = "TrackingServerProto";
+
+// API for an onboard image tracking software.
+service TrackingServerService {
+    // Set/update the current tracking status.
+    rpc SetTrackingStatus(SetTrackingStatusRequest) returns(SetTrackingStatusResponse) { option (mavsdk.options.async_type) = SYNC; }
```

Comment:
I would `SetPointTrackingStatus` and `SetRectangleTrackingStatus` instead of having to deal with parameters dependencies ("if I set mode to 'point', then I need to use the second param, and if I set it to 'rectangle', then I need to use the third param").

But that's just a personal preference :+1:

Reply:
Yes, I was considering that. Would you also do 3 requests for the commands then?

Reply:
And what about if tracking is off?